man/adoc/bpftrace.adoc

```diff
@@ -428,6 +428,12 @@ To improve the readability of big literals an underscore `_` can be used as fiel
 Integer suffixes as found in the C language are parsed by bpftrace to ensure compatibility with C headers/definitions but they're not used as size specifiers.
 `123UL`, `123U` and `123LL` all result in the same integer type with a value of `123`.
 
+These duration suffixes are also supported: `ns`, `us`, `ms`, `s`, and `min`. All get turned into integer values in nanoseconds e.g.
```

Comment:
nit: Maybe just `m` for minutes? `min` feels a little out of place contrasted with the second and subsecond abbreviation. Maybe I'm biased, since I'm used to seeing [Go style duration strings](https://pkg.go.dev/time#ParseDuration), but single letters feels more natural.

Reply:
Yeah, I thought about that too. I'm fine with `m` and I know @amscanne is also partial to Go. I'll see if anyone else feels strongly.

Reply:
`m` sounds good to me and feels more consistent with the rest.

Reply:
Although I'm partial to Go, I suspect that the reason C++ also chose `min` for duration literals is the chance of an accidental fat-finger of `ms` (which is likely to be the most common suffix).

I don't feel strongly though, and would also have a purely esthetic preference for `m` :)

I would add `h` and `d` just for completeness though, even if we don't expect them to be used as much I can imagine many scenarios where `h` is appropriate and it would be just annoying to have the others and not that.